src/octopeer-github/main/ElementEventBinding/ScrollIntoViewElementEventBinding.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Created by Mathias on 2016-05-11.
+ * Class to create ScrollIntoView events
+ */
+class ScrollIntoViewElementEventBinding implements ElementEventBinding {
+    public eventType: string = "scrollintoview";
```

Comment:
I think we first need to think about how we are going to log this event. Somehow this should still be added to the DOM however, so I think we can leave this class here for now. Maybe put a `//TODO` here :)